### ISSUE TITLE
Filenames are light grey text in choose file dialogues, hard to read (Mac)

### ISSUE THREAD

**Issue by SebastianBoell** [CompendiumNG/CompendiumNG#75]
http://compendium.open.ac.uk/bugzilla/show_bug.cgi?id=855

Michelle Bachler 2010-04-30 20:48:26 BST

Reported by Al Selvin

The folder names are legible but the filenames looks "dimmed"

[reply] [-] Comment 1 Michelle Bachler 2010-04-30 20:48:58 BST

This is a Java Mac look and feel specification.

Not sure if there is an easy way to over-ride this.

**Comment by SebastianBoell** [comment]
I will see if I can get a frien with a Mac to do some testing for us in the near future. Otherwise I suggest to announce first on Compendium Dev (and if unsuccessful on Compendium) that we are looking for two people who are willing to regularly check builds on Windows and Mac. It may be good to have some people who could invest an hour or so per week to click through the latest changes.

**Comment by maiklos** [comment]
good idea.
however I wouldn't do it until we have fully working non-developer-type installation procedure for all three major platform.
We haven't released first M1 binary download yet.

**Comment by SebastianBoell** [comment]
You are right we should not announce this too publicly if we don't have something to show for. 
- In this context I started to create a list of enhancements and bug fixes as part of M1: https://github.com/CompendiumNG/CompendiumNG/wiki/List-of-Changes-in-M1 _(I look through our complete closed issues lists and added everything from there. You should have a look and add anything else that comes to mind which I have forgotten. We can then continue to update this list while we progress.)_
- About involvement. I guess when we feel ready we should go the dev list to see if we can get two more testers involved. Ideally they can help with setting up the instructions and routines for Win and Mac. Having reliable and enthusiastic testers for Win and Mac would be good as some issues should be tested there as well before we spread the word of your efforts further into the Compendium community.
